custom_components/midea_ac_lan/__init__.py

```diff
@@ -258,3 +260,90 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     # Forward the unloading of an entry to platforms
     await hass.config_entries.async_unload_platforms(config_entry, ALL_PLATFORM)
     return True
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug(
```

Comment:
this will fire for every config entry. I think instead you want to guard against the config entry not needing to be migrated and return early if so

Reply:
Actually there are a couple of if that base don the version do or not the migration